docs/sources/mimir/configure/about-versioning.md

```diff
@@ -117,12 +117,20 @@ The following features are currently experimental:
     - `-ingester.track-ingester-owned-series`
     - `-ingester.use-ingester-owned-series-for-limits`
     - `-ingester.owned-series-update-interval`
+  - Per-ingester circuit breaking based on requests timing out or hitting per-instance limits
+    - `-ingester.circuit-breaker.enabled`
+    - `-ingester.circuit-breaker.failure-threshold`
```

Comment:
```suggestion
    - `-ingester.circuit-breaker.failure-percentage`
```

Reply:
It is `-ingester.circuit-breaker.failure-threshold-percentage`